y/metrics.go

```diff
@@ -25,67 +25,121 @@ var (
 	lsmSize *expvar.Map
 	// vlogSize has size of the value log in bytes
 	vlogSize *expvar.Map
-	// pendingWrites tracks the number of pending writes.
+	// pendingWrites tracks the number of pending writes. If no more writes happen, the value will not go to 0
 	pendingWrites *expvar.Map
 
 	// These are cumulative
 
-	// numReads has cumulative number of reads
-	numReads *expvar.Int
-	// numWrites has cumulative number of writes
-	numWrites *expvar.Int
-	// numBytesRead has cumulative number of bytes read
-	numBytesRead *expvar.Int
-	// numBytesWritten has cumulative number of bytes written
-	numBytesWritten *expvar.Int
-	// numLSMGets is number of LMS gets
+	// VLOG METRICS
+	// numReads has cumulative number of reads from vlog
+	numReadsVlog *expvar.Int
+	// numWrites has cumulative number of writes into vlog
+	numWritesVlog *expvar.Int
+	// numBytesRead has cumulative number of bytes read from VLOG
+	numBytesReadVlog *expvar.Int
+	// numBytesVlogWritten has cumulative number of bytes written into VLOG
+	numBytesVlogWritten *expvar.Int
+
+	// LSM METRICS
+	// numBytesRead has cumulative number of bytes read from LSM tree
+	numBytesReadLSM *expvar.Int
+	// numBytesLSMWritten has cumulative number of bytes written into LSM Tree
+	numBytesLSMWritten *expvar.Int
+	// numLSMGets is number of LSM gets
 	numLSMGets *expvar.Map
+	// numBytesCompactionWritten is the number of bytes written in the lsm tree due to compaction
+	numBytesCompactionWritten *expvar.Int
 	// numLSMBloomHits is number of LMS bloom hits
 	numLSMBloomHits *expvar.Map
-	// numGets is number of gets
+
+	// DB METRICS
+	// numGets is number of gets -> Number of get requests made
 	numGets *expvar.Int
-	// numPuts is number of puts
+	// number of get queries in which we actually get a result
+	numGetsWithResults *expvar.Int
```

Comment:
i am assuming you are trying to find how many out of numGets were in badger ? should this instead be numKeyNotFound or similar ?